docs/features/bonding-quick-start.md

```diff
@@ -0,0 +1,140 @@
+
+Two modes are supported:
+
+- [Broadcast](./socket-groups.md#1-broadcast) - In *Broadcast* mode, data is sent redundantly over all the member links in a group. If one of the links fails or experiences network jitter and/or packet loss, the missing data will be received over another link in the group. Redundant packets are simply discarded at the receiver side.
+
+- [Main/Backup](./bonding-main-backup.md) - In *Main/Backup* mode, only one (main) link at a time is used for data transmission while other (backup) connections are on standby to ensure the transmission will continue if the main link fails. The goal of Main/Backup mode is to identify a potential link break before it happens, thus providing a time window within which to seamlessly switch to one of the backup links.
+
+TODO: Enabled with ...
+
+## 2. Additional Documentation and Code Examples
+
+- [SRT Connection Bonding](./bonding-intro.md) - Introduction to Connection Bonding. Description of group (bonded) connections.
+- [SRT Socket Groups](./socket-groups.md) - Description of socket groups in SRT. Here you will also find the information regarding the `srt-test-live` application for testing Connection Bonding.
+- [SRT Connection Bonding: Main/Backup](./bonding-main-backup.md) - Main/Backup mode description.
+- Check also [SRT API](../API/API.md) and [SRT API Functions](../API/API-functions.md#socket-group-management) documentation for the bonding related updates.
+- Code examples: simple [client](../../examples/test-c-client-bonding.c) and [server](../../examples/test-c-server-bonding.c) implementation.
+
+## 3. Quick Start Guidelines
+
+### 3.1. Transition to a Bonded Connection
+
+For the end-user of the SRT library, operations on a group of bonded sockets should be very much like on a single SRT socket. Important thing is that application needs to regularly check for broken sockets in a group and reconnect those links that are broken. See [SRT API Functions](../API/API-functions.md#socket-group-management) that can be used with a socket group.
+
+**On the listener side (pseudocode):**
+
+* Create a socket: `sock_id = srt_create_socket()`,
+* `srt_bind(sock_id, ..)` to an interface (as usual for a listener socket),
+* `srt_setsockflag(sock_id, SRTO_GROUPCONNECT, &(int yes = 1), sizeof yes)` (by allowing group connections your app confirms being group aware),
+* Listen for incoming connections: `srt_listen(sock_id, ...)` (as usual),
+* Accept a bonded connection with `accepted_group_id = srt_accept(sock_id, ...)` will return the ID of the accepted socket group once the first socket group member establishes a connection.
```

Comment:
```suggestion
* Accept a bonded connection with `accepted_group_id = srt_accept(sock_id, ...)`. This will return the ID of the accepted socket group once the first socket group member establishes a connection.
```